test/tools/WebListener/Controllers/RetryController.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.Extensions.Primitives;
+using mvc.Models;
+
+namespace mvc.Controllers
```

Comment:
CodeFactor - Namespace names begin with an upper-case letter
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1300.md

Reply:
This will cause unnecessary changes across the WebListener project. I prefer not fixing this.